============================= test session starts ==============================
platform linux -- Python 3.9.18, pytest-7.4.2, pluggy-1.3.0
rootdir: /workspace
collected 10 items / 9 deselected / 1 selected
tests/test_a_setup.py F
======================
Output of autograder
======================
# of tests:	0/1 tests passed
Overall score:	0/100
======================
FAIL - REQUIRED (automatic zero) test_setup
======================
=================================== FAILURES ===================================
__________________________________ test_setup __________________________________
>           assert fn in files, f"Please create {fn}"
E           AssertionError: Please create jwt9656_q1.pdf
E           assert 'jwt9656_q1.pdf' in ['.git', '.gitignore', 'README.md', 'free_response', 'hw2.ipynb', 'models', ...]
tests/test_a_setup.py:26: AssertionError
=========================== short test summary info ============================
FAILED tests/test_a_setup.py::test_setup - AssertionError: Please create jwt9...
======================= 1 failed, 9 deselected in 3.58s ========================